install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -182,6 +182,12 @@ type SpiceDBConfig struct {
 	SecretRef string `json:"secretRef"`
 }
 
+type RedisConfig struct {
+	Address   string `json:"address,omitempty"`
+	Username  string `json:"username,omitempty"`
+	SecretRef string `json:"secretRef,omitempty"`
```

Comment:
why ist this called secretref here and password further down the stack? Can we align on one way to name this property?

Reply:
They refer to two different things. Here we refer to a Kubernetes secret that needs to be created beforehand. This secret needs to have a password entry that contains the password. Only this entry is then passed to the deployment as environment variable.

This reminded me that we also need to pass the secret to ws-manager-bridge. I forgot that :facepalm: